downstream/assemblies/hub/assembly-populate-container-registry.adoc

```diff
@@ -23,12 +23,8 @@ You must follow a specific workflow to populate your {PrivateHubName} container
 ====
 Image manifests and filesystem blobs were both originally served directly from `registry.redhat.io` and `registry.access.redhat.com`. 
 As of 1 May 2023, filesystem blobs are served from `quay.io` instead. 
-To avoid problems pulling container images, enable outbound connections to the following hostnames:
 
-* `cdn.quay.io`
-* `cdn01.quay.io`
-* `cdn02.quay.io`
-* `cdn03.quay.io`
+* Ensure that the link:https://access.redhat.com/documentation/en-us/red_hat_ansible_automation_platform/2.4/html/red_hat_ansible_automation_platform_planning_guide/ref-network-ports-protocols_planning[Network ports and protocols] listed in _Table 5.10. Execution Environments (EE)__ are available to avoid problems pulling container images.
```

Comment:
```suggestion
* Ensure that the link:{BaseURL}/red_hat_ansible_automation_platform_planning_guide/ref-network-ports-protocols_planning[Network ports and protocols] listed in _Table 5.10. Execution Environments (EE)_ are available to avoid problems pulling container images.
```